Unreal/CarlaUE4/Plugins/Carla/Source/Carla/Settings/CarlaSettingsDelegate.cpp

```diff
@@ -159,13 +165,17 @@ void UCarlaSettingsDelegate::CheckCarlaSettings(UWorld *world)
 
 void UCarlaSettingsDelegate::LaunchLowQualityCommands(UWorld *world) const
 {
+  if (!world)
+  {
+    return;
+  }
+
```

Comment:
Thanks for adding this! Not sure under what circumstances it would get popped, but I definitely appreciate that the function implementations are now consistent with each other.

Reply:
Yes, I think it is more consistent that way. Thanks.